src/services/enum-utilities.ts

```diff
@@ -389,9 +389,11 @@ export default class EnumUtilities {
    * @param type the filing type to convert
    * @param agmYear the AGM Year to be appended to the filing name (optional)
    * @param subType the filing subtype (optional)
+   * @param filingStatus the filing status (optional)
    * @returns the filing name
    */
-  static filingTypeToName (type: FilingTypes, agmYear = null as string, subType: FilingSubTypes = null): string {
+  static filingTypeToName (type: FilingTypes, agmYear = null as string, subType: FilingSubTypes = null,
+    filingStatus: FilingStatus = null): string {
```

Comment:
Move `): string {` to a new line and see if you like the style better.

Reply:
Updated, thanks